src/sqlitestorage.h

```diff
@@ -672,9 +672,13 @@ class MKCAL_EXPORT SqliteStorage : public ExtendedStorage
 #define SELECT_COMPONENTS_BY_LAST_MODIFIED_AND_NOTEBOOK \
 "select * from Components where DateLastModified>=? and DateCreated<? and Notebook=? and DateDeleted=0"
 #define SELECT_COMPONENTS_BY_DELETED \
-"select * from Components where DateDeleted>=? and DateCreated<?"
+"select * from Components where DateDeleted<>0"
```

Comment:
3) The same comment applies to SELECT_COMPONENTS_BY_DELETED_AND_NOTEBOOK below.
My question is similar to (2).